libraries/liblmdb/scale.sh

```diff
@@ -0,0 +1,32 @@
+set -e
+  
+DATASET_NAMES=(
+    "gmm_k100_200M_uint64"
+    "gmm_k100_400M_uint64"
+    "gmm_k100_600M_uint64"
+    "gmm_k100_800M_uint64"
+)
+
+ROOT=$1
+DATA_PATH=$2
+echo "${ROOT}"
+
+echo "Start LMDB Scalability Test"
+
+for ((i = 0; i < ${#DATASET_NAMES[@]}; i++)) do
+    mkdir -p ${ROOT}/temp/lmdb/${dataset_name}
+    mkdir -p ${ROOT}/storage/lmdb/${dataset_name}
+    dataset_name="${DATASET_NAMES[$i]}"
+    echo ">>> ${dataset_name} ${j}"
+    export RAYON_NUM_THREADS=128
+    SECONDS=0
+    make kv_build && ./kv_build \
```

Comment:
Move `make kv_build` to before this for-loop. It wouldn't affect the build time much on second run, but we shouldn't measure it anyway.